db/dev-or-prod-migrations.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="replace-dots-with-hyphens" author="sanyavertolet" context="">
+        <sql>
+            update save_cloud.`user` set name = replace (name, '.', '-');
```

Comment:
I don't think we should do it right now, because it can break login for already existing users. We simply take login from an OAuth provider, and we don't support overriding it just yet

Reply:
Oh, sure, it is too early for users :)

Reply:
Fixed